composer.json

```diff
@@ -16,6 +16,8 @@
     },
     "require": {
         "php": "^7.4",
+        "league/flysystem": "^2.0@dev",
```

Comment:
We can't use the `dev` version in the production. Can you make issue to track the stable release?

Reply:
Ok

Reply:
We must use version 2.0 of flysystem, it is the cleanest and corrects many errors.

Reply:
> We must use version 2.0 of flysystem, it is the cleanest and corrects many errors.

It used in this PR.